docs/usage.md

```diff
@@ -1,153 +1,306 @@
+- [Custom configuration](#custom-configuration)
+  - [Updating containers (advanced users)](#updating-containers--advanced-users-)
+  - [nf-core/configs](#nf-core-configs)
+  * [Run sentieon](#run-sentieon)
+  * [Azure Resource Requests](#azure-resource-requests)
+  * [Running in the background](#running-in-the-background)
+  * [Nextflow memory requirements](#nextflow-memory-requirements)
+
+# Introduction
+
+nf-core/raredisease is a bioinformatics best-practice analysis pipeline to call, annotate and score variants from WGS/WES of rare disease patients. The pipeline is built using nextflow.
+
+# Prerequisites
+
+1. Install Nextflow (>=22.10.1) using the instructions [here.](https://nextflow.io/docs/latest/getstarted.html#installation)
+2. Install one of the following technologies for full pipeline reproducibility: Docker, Singularity, Podman, Shifter or Charliecloud.
+   > Almost all nf-core pipelines give you the option to use conda as well. However, some tools used in the raredisease pipeline do not have a conda package so we do not support conda at the moment.
+
+# Run nf-core/raredisease with test data
+
+Before running the pipeline with your data, we recommend running it with the test dataset available [here](https://github.com/nf-core/test-datasets/tree/raredisease).
+
+> You do not need to download the data as the pipeline is configured to fetch that data automatically for you when you use the test profile.
+
+Run the following command, where YOURPROFILE is the package manager you installed on your machine. For example, `-profile test,docker` or `-profile test,singularity`:
 
-> _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
+```
+nextflow run nf-core/raredisease \
+    -revision dev -profile test,<YOURPROFILE> \
+    --outdir <OUTDIR>
+```
 
-## Introduction
+> Check [nf-core/configs](https://github.com/nf-core/configs/tree/master/conf) to see if a custom config file to run nf-core pipelines already exists for your institute. If so, you can simply use `-profile test,<institute>` in your command. This enables the appropirate package manager and sets the appropriate execution settings for your machine.
+> NB: The order of profiles is important! They are loaded in sequence, so later profiles can overwrite earlier profiles.
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+The above command downloads the pipeline from github, caches it, and tests it on the test dataset.
 
-## Samplesheet input
+> When you run the command again, it will fetch the pipeline from cache even if a latest version of the pipeline is available. To make sure that you're running the latest version of the pipeline, update the cached version of the pipeline by including `-latest` in the command.
```

Comment:
```suggestion
> When you run the command again, it will fetch the pipeline from cache even if a more recent version of the pipeline is available. To make sure that you're running the latest version of the pipeline, update the cached version of the pipeline by including `-latest` in the command.
```